allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py

```diff
@@ -2,42 +2,10 @@
 from overrides import overrides
 
 import spacy
-import ftfy
 from transformers.tokenization_bert import BasicTokenizer as BertTokenizer, _is_punctuation
 
-from allennlp.common.util import get_spacy_model
 from allennlp.data.tokenizers.token import Token
 from allennlp.data.tokenizers.tokenizer import Tokenizer
-from allennlp.data.token_indexers.openai_transformer_byte_pair_indexer import text_standardize
-
-
-@Tokenizer.register("openai")
-class OpenAIPreTokenizer(Tokenizer):
-    """
-    For OpenAI transformer
-    This is used to split a sentence into words.
-    Then the ``OpenaiTransformerBytePairIndexer`` converts each word into wordpieces.
-    """
-
-    def __init__(self, language: str = "en_core_web_sm") -> None:
-        self.spacy = get_spacy_model(language, False, False, False)
-
-    @staticmethod
-    def _standardize(text):
-        return text_standardize(ftfy.fix_text(text))
-
-    @overrides
-    def batch_tokenize(self, texts: List[str]) -> List[List[Token]]:
-        standardized_sentences = [self._standardize(sentence) for sentence in texts]
-        return [
-            _remove_spaces(tokens)
-            for tokens in self.spacy.pipe(standardized_sentences, n_threads=-1)
-        ]
-
-    @overrides
-    def tokenize(self, text: str) -> List[Token]:
-        # This works because our Token class matches spacy's.
-        return _remove_spaces(self.spacy(self._standardize(text)))
 
 
 @Tokenizer.register("bert-basic")
```

Comment:
We'll want to remove the old BERT ones too, which will let us remove this file entirely.  That can be done as soon as #3634 is merged.

Reply:
let's do that in pieces, so we can keep it tracked.

Reply:
I agree, just noting.

Reply:
I might be missing something but I'm not touching the `pre_tokenizer` class in #3634. So it shouldn't be relevant?

Reply:
#3634 takes the logic that was in the `BertIndexer` and related classes and implements it for all transformers in the huggingface library.  This makes the `Bert*` classes (including the one mentioned here) obsolete, so they can be removed.